From: Jeremy Bicha <jbicha@debian.org>
Date: Wed, 26 Sep 2018 20:13:06 -0400
Subject: skip texture test

This test fails on hppa, mipsel, powerpc, ppc64 and s390x.

See the build logs at
https://buildd.debian.org/status/logs.php?pkg=mutter&ver=3.30.0-2

I don't know why this test works in the clutter-1.0 package.

Bug: https://gitlab.gnome.org/GNOME/mutter/issues/324

Applied-upstream: no
---
 clutter/tests/conform/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/clutter/tests/conform/Makefile.am b/clutter/tests/conform/Makefile.am
index 9cef08b..0f4681f 100644
--- a/clutter/tests/conform/Makefile.am
+++ b/clutter/tests/conform/Makefile.am
@@ -55,7 +55,6 @@ deprecated_tests = \
 	behaviours \
 	group \
 	rectangle \
-	texture \
 	$(NULL)
 
 test_programs = $(actor_tests) $(general_tests) $(classes_tests) $(deprecated_tests)
